tests/providers/docker/hooks/test_docker.py

```diff
@@ -36,7 +36,7 @@
 TEST_CONN = {"host": "some.docker.registry.com", "login": "some_user", "password": "some_p4$$w0rd"}
 MOCK_CONNECTION_NOT_EXIST_MSG = "Testing connection not exists"
 MOCK_CONNECTION_NOT_EXISTS_EX = AirflowNotFoundException(MOCK_CONNECTION_NOT_EXIST_MSG)
-HOOK_LOGGER_NAME = "airflow.providers.docker.hooks.docker.DockerHook"
+HOOK_LOGGER_NAME = "airflow.task.hooks.airflow.providers.docker.hooks.docker.DockerHook"
```

Comment:
I guess we also need full test for this PR, because it might affect different providers

Reply:
No need-> "full tests" is only when you run the whole matrix/python/databases. 
Any change in "core" (so anything providers can depend on)  is triggering all provider tests anyway.

Reply:
But feel free :)

Reply:
BTW. It's nicely explained in selective checks output (selective checks tell exactly what decisions are made along the way): 

https://github.com/apache/airflow/pull/34964/checks#step:11:69

<img width="759" alt="Screenshot 2023-10-24 at 00 31 04" src="https://github.com/apache/airflow/assets/595491/e2974c6d-2b76-4320-9218-d9eb7f2dc5e4">

Reply:
(Having changed tests there is just being cautious).

Reply:
By full test, you mean adding one test for each hooks and operators, right ?

I considered it, especially because any folder / file renaming could cause a breaking change for users with advanced logging config. I haven't done it because it takes time. But I can certainly add those tests

Reply:
It was just about running "all testss" -> this is wheat setting "full tests needed" forces.